starfish/image/_filter/gaussian_high_pass.py

```diff
@@ -17,30 +17,37 @@
 
 
 class GaussianHighPass(FilterAlgorithmBase):
+    """
+    Applies a Gaussian high pass filter to the ImageStack. This is useful to remove cellular
+    autofluorescence, which is typically low frequency.
+
+    This filter works by subtracting a Gaussian low pass filter defined in
```

Comment:
This filter works by subtracting a Gaussian low pass filtered version of the input image from the input image itself. This low pass filter is defined in